lib/Importer/ProtobufLoader.cpp

```diff
@@ -320,6 +372,12 @@ void ProtobufLoader::setupLoader(llvm::ArrayRef<const char *> tensorNames,
           placeholder, createAndRegisterPlaceholder(tensorNames[i], T));
       inputVarsByName_.try_emplace(tensorNames[i], placeholder);
     }
+    // set model loader precision config if node precision info is provided
```

Comment:
Sorry for the nits on grammar but we try to be strict 🙂 
```suggestion
    // Set model loader precision config if node precision info is provided.
```

Reply:
Done